app/controllers/auth_key_pair_cloud_controller.rb

```diff
@@ -149,22 +149,20 @@ def create_finished
       }, :error)
     end
 
-    @breadcrumbs.pop if @breadcrumbs
+    @breadcrumbs&.pop
     session[:edit] = nil
     flash_to_session
-    javascript_redirect :action => "show_list"
+    javascript_redirect(:action => "show_list")
   end
 
   # delete selected auth key pairs
   def delete_auth_key_pairs
     assert_privileges("auth_key_pair_cloud_delete")
-    key_pairs = []
-
-    if @lastaction == "show_list" || (@lastaction == "show" && @layout != "auth_key_pair_cloud")
-      key_pairs = find_checked_records_with_rbac(ManageIQ::Providers::CloudManager::AuthKeyPair)
-    else
-      key_pairs = [find_record_with_rbac(ManageIQ::Providers::CloudManager::AuthKeyPair, params[:id])]
-    end
+    key_pairs = if @lastaction == "show_list" || (@lastaction == "show" && @layout != "auth_key_pair_cloud")
```

Comment:
@mzazrivec this can be simplified to 
```ruby
key_pairs = find_records_with_rbac(ManageIQ::Providers::CloudManager::AuthKeyPair, checked_or_params)
```

`find_checked_records_with_rbac` uses `find_checked_items` method to find items checked in GTL.

The same can be achieved by using `checked_or_params`, which also tries to find the records id in `params[:id]`, if it is present.

Both of the methods do return an array of instances, that you can get by using `find_records_with_rbac` and you don't need to wrap the `find_record_with_rbac` into an array.

Reply:
OK, sure -- done. Although this is not exactly a rubocop fix.